icu4c/source/test/intltest/caltest.cpp

```diff
@@ -5634,8 +5634,10 @@ void CalendarTest::TestFirstDayOfWeek() {
     // ssh => ssh_Arab_AE => Saturday
     verifyFirstDayOfWeek("ssh", UCAL_SATURDAY);
     // wbl_Arab => wbl_Arab_AF => Saturday
-    verifyFirstDayOfWeek("wbl-Arab", UCAL_SATURDAY);
-
+	if (!logKnownIssue("CLDR-17907", "wbl-Arab returns wrong first day of the week, probably caused by wrong Likely Subtag algo")) {
+       verifyFirstDayOfWeek("wbl-Arab", UCAL_SATURDAY);
+    }    
```

Comment:
I agree with [Mark's ticket comment](https://unicode-org.atlassian.net/browse/ICU-22869?focusedCommentId=176427). Please remove this test case altogether.

Reply:
Gone